### PR TITLE
Expose createOffsetTableIfNotExists for testing purposes

### PR DESCRIPTION
**Pull request by patriknw**
Need this for doc examples.

### CODE REVIEW

File: akka-projection-cassandra/src/main/scala/akka/projection/cassandra/internal/CassandraProjectionImpl.scala [hunk 1]
```diff
@@ -73,11 +76,12 @@ import akka.stream.scaladsl.Source
...
     settingsOpt: Option[ProjectionSettings],
     handler: Handler[Envelope])
-    extends Projection[Envelope] {
+    extends javadsl.CassandraProjection[Envelope]
+    with scaladsl.CassandraProjection[Envelope] {
   import CassandraProjectionImpl._
   import HandlerRecoveryImpl.applyUserRecovery
 
-  override def withSettings(settings: ProjectionSettings): Projection[Envelope] = {
+  override def withSettings(settings: ProjectionSettings): CassandraProjectionImpl[Offset, Envelope] = {
```
Comment (renatocaval) [inline akka-projection-cassandra/src/main/scala/akka/projection/cassandra/internal/CassandraProjectionImpl.scala:84]:
```suggestion
  override def withSettings(settings: ProjectionSettings): CassandraProjection[Offset, Envelope] = {
```

Comment (patriknw) [reply akka-projection-cassandra/src/main/scala/akka/projection/cassandra/internal/CassandraProjectionImpl.scala:84]: that's a trick for javadsl/scaladsl
withSettings is overridden in the javadsl.CassandraProjection and scaladsl.CassandraProjection, and that is what the end user will see

**Comment by patriknw** [comment]
Strange, I don't see that compilation error locally, but I'll rename one of them anyway.

**Comment by renatocaval** [comment]
> Strange, I don't see that compilation error locally, but I'll rename one of them anyway.

So, there is a compilation error. 

I stopped on the first one, the logging one. That's what flaky tests do to us, we  (I) stop looking for other errors and restart the build.

File: akka-projection-cassandra/src/main/scala/akka/projection/cassandra/internal/CassandraProjectionImpl.scala (hunk 1 above)
Comment (renatocaval) [reply akka-projection-cassandra/src/main/scala/akka/projection/cassandra/internal/CassandraProjectionImpl.scala:84]: Ok, I see. Not a big issue.

**Pull request merged**